pkg/tiller/kind_sorter.go

```diff
@@ -123,5 +123,9 @@ func (k *kindSorter) Less(i, j int) bool {
 	if !ok {
 		return true
 	}
+	if first == second {
+		// same kind so sub sort alphanumeric
+		return a.name < b.name
+	}
```

Comment:
Yes, I think it's fine to let it slide here. It's not worse than the current behavior.
The only question is how to remember to come back and attend to it, if you believe that it's worth doing so.